src/hotspot/share/runtime/interfaceSupport.inline.hpp

```diff
@@ -490,6 +490,7 @@ extern "C" {                                                         \
     JavaThread* thread=JavaThread::thread_from_jni_environment(env); \
     ThreadInVMfromNative __tiv(thread);                              \
     debug_only(VMNativeEntryWrapper __vew;)                          \
+    debug_only(ResetNoHandleMark __rnhm;)                            \
```

Comment:
Your fix for JDK-8259374 adds a ResetNoHandleMark to
ThreadInVMfromNative so new L493 isn't needed.

Reply:
Oh you're right.  I'd missed this in the original.  I'll remove this after I checkin the first one.